src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,81 @@
+.product-card {
+  margin-bottom: 48px;
+  width: 200px;
+  height: 406px;
+  margin-right: $gap;
+  background-color: white;
+  border: 1px solid #f3f3f3;
+  border-radius: 6px;
+  padding: 32px 16px 0;
+
+  &__img {
+    display: block;
+    background-repeat: no-repeat;
+    background-size: 160px 134px;
+    height: 134px;
+    width: 160px;
+    margin: 0 auto;
+  }
+  &__name {
+    font-weight: $medium_font;
+    color: $primary-color;
+    line-height: 18px;
+    margin-top: 40px;
+    font-size: 12px;
+  }
+  &__code {
+    color: #616070;
+    margin-top: 4px;
+    font-weight: $small_font;
+    font-size: 10px;
+  }
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
```

Comment:
same with this, it should be conneted to catalog's element, not to card block itself